res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -1572,3 +1573,7 @@ TraktorS4MK2.onLoopSizeChanged = function(value, group, key) {
 TraktorS4MK2.loopSizeSet = function(group) {
   TraktorS4MK2.onLoopSizeChanged(engine.getValue(group, "beatloop_size"), group);
 }
+
+TraktorS4MK2.onRecordingChanged = function(group) {
+  TraktorS4MK2.outputChannelCallback(engine.getValue("[Recording]", "status"), "[Recording]", "status");
```

Comment:
This function gets passed the parameters `value, group, control`. Use the `value` parameter instead of making a redundant call to `engine.getValue`